data-esnext.js

```diff
@@ -1686,6 +1686,26 @@ exports.tests = [
     }
   ]
 },
+{
+  name: 'template literal revision',
+  category: 'candidate (stage 3)',
+  significance: 'small',
+  link: 'https://github.com/tc39/proposal-template-literal-revision',
+  exec: function() {/*
+    function fn(strings, a, b) {
+      return typeof strings[0] === "undefined" &&
+        typeof strings[1] === "undefined" &&
+        typeof strings[2] === "undefined" &&
```

Comment:
Do you mean none of the parameters should be checked for undefined, just [1] and [2] (which make the test pass on Firefox), or just [2] (which is the \u{55})?